test.py

```diff
@@ -9,18 +9,26 @@
 import tempfile
 import time
 
-
```

Comment:
This is not a regression. Regression is a serious word that should not be diluted by nitpicks over a blank linke.